src/diffusers/utils/hub_utils.py

```diff
@@ -393,6 +401,21 @@ def _get_model_file(
             )
 
 
+def _fetch_remapped_cls_from_config(config, old_class):
```

Comment:
I think this can live in `modeling_utils.py`?

Reply:
Used in the `configuration_utils.py`, too.